### PR TITLE
Change ops-runner and some cron scripts to Use MetricSender

### PR DESCRIPTION
**Pull request by zgalor**
Depends on #1747

### CODE REVIEW

**Comment by zgalor** [comment]
@twiest PTAL

**Comment by zgalor** [comment]
@joelddiaz PTAL

**Review by twiest — APPROVED** [review]
@zgalor This looks great!

This is ready to merge right after our holiday freeze is lifted.

**Pull request merged**
